apis/stacks/v1alpha1/types.go

```diff
@@ -305,6 +305,14 @@ type StackStatus struct {
 	ControllerRef                     *corev1.ObjectReference `json:"controllerRef,omitempty"`
 }
 
+// PackageMetadataSpec defines metadata about the stack application
+// and package contents
+type PackageMetadataSpec struct {
+	DesignVersion string `json:"designVersion,omitempty"`
```

Comment:
What about `metaVersion`? o: it's clearly not for the content because of `meta`, but it doesn't tie us directly to the design. And it can encompass more than just the `app.yaml` because `meta` isn't specific.

Reply:
i'd like to put a vote in for `formatVersion`, since it feels like that would cover the full spectrum of app metadata and package matadata versioning...the entire package **format version** :)

Reply:
`formatVersion` was on my original poll, so it has my blessing. I think it, like `metaVersion`, conveys that there is more to this than the app.yaml file.  
`designVersion` has the weakness that it alludes to `design` document (compared to a specification).
`apiVersion` also works, but I think it is a mixed blessing that both Helm and Kustomize already use the same name.

I'll update the PR to use `formatVersion` and see how that goes. :-)